### PR TITLE
manual: Add missing section on tarball-ttl

### PR DESCRIPTION
**Pull request by nh2**

### CODE REVIEW

**Comment by edolstra** [comment]
See also https://github.com/NixOS/nix/pull/2882.

File: doc/manual/command-ref/conf-file.xml [hunk 1]
```diff
@@ -865,6 +865,26 @@ requiredSystemFeatures = [ "kvm" ];
...
+    <listitem>
+
+      <para>This option defines the number of seconds that a
+      downloaded tarball is cached in <filename>~/.cache/nix/tarballs/</filename>.
+      </para>
+
+      <para>This is most relevant for fetching tarballs that change remotely
+      via <envar>NIX_PATH</envar> or <function>fetchGit</function>.
+      Using <option>--option tarball-ttl 0</option> forces nix to re-fetch
+      these tarballs.</para>
```
Comment (edolstra) [inline doc/manual/command-ref/conf-file.xml]: This is not enitirely correct. TTL=0 doesn't force a re-fetch, it forces Nix to check whether the tarball is still up-to-date using the ETag.

Comment (grahamc) [reply doc/manual/command-ref/conf-file.xml]: I incorporated much of this text, and this feedback in https://github.com/NixOS/nix/pull/2882/files

**Comment by teto** [comment]
can be closed I guess since other PR got merged

**Pull request closed without merging**